### PR TITLE
Issue-35 Table Rules and Triggers

### PR DESCRIPTION
**Pull request by Netravati1205**
Simple Rules - for all character based rules are added.

Here is the verifier report for windows and IBMi
[Verifier_Test_Report.txt](https://github.com/lansa/db-regression/files/14029037/Verifier_Test_Report.txt)

### CODE REVIEW

**Review by robe070 — COMMENTED** [review]
Looks good

**Comment by Netravati1205** [comment]
Added changes for Numeric based rules.

File: Partition (WBP)/Functions/VI0035A.VTI0035.fun.yml [hunk 1]
```diff
@@ -514,6 +514,7 @@
     Clr_List Named(#wl_VTTEMP_Function)
     
     #Com_Owner.mthAddFunction( VI0035C )
+    #Com_Owner.mthAddFunction( VI0035D )
```
Comment (robe070) [inline Partition (WBP)/Functions/VI0035A.VTI0035.fun.yml:524]: This needs to be
```
If (#LF_VTTEMP_Skip_Test_Case = False)
#Com_Owner.mthAddFunction( VI0035D )
Else
#LF_VTTEMP_Number_Of_Skipped_Test_Case += 1
Endif
```

Comment (Netravati1205) [reply Partition (WBP)/Functions/VI0035A.VTI0035.fun.yml:524]: Okay , Will add it as above

**Comment by robe070** [comment]
Test case values look to be quite thorough. Good work.

**Comment by Netravati1205** [comment]
Report for Date based rules test cases.
[Verifier_Test_Report_DateRules.txt](https://github.com/lansa/db-regression/files/14048029/Verifier_Test_Report_DateRules.txt)

File: Partition (WBP)/Functions/VI0035A.VTI0035.fun.yml (hunk 1 above)
Comment (Netravati1205) [reply Partition (WBP)/Functions/VI0035A.VTI0035.fun.yml:524]: This has been incorporated with below commit.

**Comment by Netravati1205** [comment]
Report for Remaining rules other than quick rules.
[Verifier_Test_Report_other_remaining rules.txt](https://github.com/lansa/db-regression/files/14058598/Verifier_Test_Report_other_remaining.rules.txt)

**Comment by Netravati1205** [comment]
All Rules related changes Verifier report
[Verifier_Test_Report_All_rules.txt](https://github.com/lansa/db-regression/files/14061738/Verifier_Test_Report_All_rules.txt)

**Pull request merged**